### PR TITLE
Ar 87 immersive image caption

### PR DESCRIPTION
**Pull request by dskamiotis**
## Why are you doing this?

Captions and credits were not displaying on Immersive articles. This PR checks if its an immersive header and adds the appropriate caption and credit to match the dot com behaviour and styling.

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

**Comment by JamieB-gu** [comment]
Thanks for dropping the `yarn.lock`! I think it would be best if @webb04 or @frankie297 gave the final review on this as I was involved in several of the changes.

File: src/components/immersiveCaption.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { FC, ReactElement } from 'react';
+import { Item } from 'item';
+import { MainMedia, MainMediaKind } from 'headerMedia';
+import { Option, OptionKind } from '@guardian/types/option';
+import { renderCaption } from './figCaption'
+import { Format } from "../format";
```
Comment (webb04) [inline src/components/immersiveCaption.tsx:6]:
``` suggestion
import { renderCaption } from 'components/figCaption'
import { Format } from "format";
```

File: src/components/immersiveCaption.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { Display } from '@guardian/types/Format';
+
+interface Props {
+    item: Item;
+}
+
+const captionHeadingStyles = css`
+    ${textSans.xsmall()}
+    color: ${neutral[46]};
+    margin: 0 0 ${remSpace[3]};
```
Comment (webb04) [inline src/components/immersiveCaption.tsx:20]:
```suggestion
    margin: ${remSpace[4]} 0;
```

File: src/components/immersiveCaption.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React, { FC, ReactElement } from 'react';
+import { Item } from 'item';
+import { MainMedia, MainMediaKind } from 'headerMedia';
+import { Option, OptionKind } from '@guardian/types/option';
+import { renderCaption } from 'components/figCaption'
+import { Format } from "format";
```
Comment (webb04) [inline src/components/immersiveCaption.tsx:6]:
```suggestion
import { Format } from '@guardian/types/Format';
```

Comment (webb04) [reply src/components/immersiveCaption.tsx:6]: This looks like it's an error in the CI

Comment (dskamiotis) [reply src/components/immersiveCaption.tsx:6]: had trouble pushing the new changes

Comment (dskamiotis) [reply src/components/immersiveCaption.tsx:6]: Apparently there is a problem with the format:
TS7016: Could not find a declaration file for module 'format'. '/home/runner/work/apps-rendering/apps-rendering/node_modules/format/format.js' implicitly has an 'any' type.
Is this due to the format file change Jamie did?

**Pull request merged**